cobc/parser.y

```diff
@@ -1369,6 +1369,16 @@ setup_program (cb_tree id, cb_tree as_literal, const enum cob_module_type type,
 
 	/* Initalize default COLLATING SEQUENCE */
 	default_collation = build_colseq (cb_default_colseq);
+	switch (cb_default_colseq) {
```

Comment:
something is wrong here, as this is later adjusted in `object_computer_sequence`;
from a quick glance you want to adjust the default of `alphanumeric_collation` and `national_collation`

Reply:
I'd suggest to init both `alphanumeric_collation` and `national_collation` in `program_init_without_program_id` and `setup_program`, then later use it directly.